samples/csharp_webapi/13.core-bot/Schemas/Microsoft.IDialog.schema

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
and this one too?

Reply:
OK so the schema (and the  project file updates) come from the NuGet package.